op-batcher/batcher/driver.go

```diff
@@ -831,10 +831,12 @@ func (l *BatchSubmitter) publishTxToL1(ctx context.Context, queue *txmgr.Queue[t
 	}
 	l.Metr.RecordLatestL1Block(l1tip)
 
+	_, params := l.throttleController.Load()
+	isThrottling := params.Intensity > 0.0
```

Comment:
Instead of `0.0` as the comparator here, we could use a variable defined in the CLI flags (which defaults to  `0.0`). What are your thoughts?

On the one hand it's good to hardcode and have consistent behavior across the OPStack / superchain, and keep the config from bloating. On the other hand, we can leave the door open to changing the param easily in future by making it configurable. It's always nice, too, to be able to disable or undo a change like this in case it has unintended or unforeseen consequences.

Reply:
Personally, I wouldn't expose a config here. I think `> 0.0` is the behavior we want: the new throttling features allow for communicating the throttling intensity, but anything greater than zero is still throttling. This interpretation is also used [here](https://github.com/ethereum-optimism/optimism/blob/57e2a302363d35672db8445281979d14d2a506d5/op-batcher/batcher/driver.go#L682).

I've refactored a bit to collect this code in one place.